Orange/data/table.py

```diff
@@ -1727,12 +1732,16 @@ def transpose(cls, table, feature_names_column="",
         self.X = table.X.T
         if feature_names_column:
             names = [str(row[feature_names_column]) for row in table]
+            progress_callback(0.1)
             names = get_unique_names_duplicates(names)
+            progress_callback(0.3)
             attributes = [ContinuousVariable(name) for name in names]
         else:
             places = int(np.ceil(np.log10(n_cols))) if n_cols else 1
             attributes = [ContinuousVariable(f"{feature_name} {i:0{places}}")
                           for i in range(1, n_cols + 1)]
+        progress_callback(0.4)
```

Comment:
I believe the correct value here is 0.418. Please check your calculations again.